### ISSUE TITLE
Details Dropdown in Wiki Sidebar

### ISSUE THREAD

**Issue by bahadirgezer** [bounswe/bounswe2023group8#31]
I believe that the dropdown menu in the sidebar should be removed, here is why: 

Drop-down menus have been a popular design element for websites for quite some time. However, there are valid reasons to argue that drop-down menus in the sidebar are not required, slow down access times, and can be bad for user experience.

Firstly, drop-down menus in the sidebar are not always necessary. They are often used to organize large amounts of information, but they can easily become overwhelming and difficult to navigate. In fact, studies have shown that users tend to avoid using drop-down menus in favor of more visible navigation options. This means that implementing drop-down menus in the sidebar can be unnecessary and redundant.

Secondly, drop-down menus can slow down access times. Users have to click on the menu to see the options, which adds an extra step to the navigation process. This may not seem like a big deal, but it can add up over time and significantly slow down the user experience. In addition, drop-down menus can also be difficult to use on touch devices, making them inaccessible to a large portion of the user base.

Lastly, drop-down menus can be bad for user experience. They can be confusing and frustrating for users who are not familiar with the website's navigation structure. Users may not be able to find what they are looking for, which can lead to increased bounce rates and decreased user engagement. Additionally, drop-down menus can be difficult to use for users with disabilities, such as those who use screen readers.

In conclusion, drop-down menus in the sidebar are not required and can be detrimental to user experience. While they may seem like a useful design element, they can actually slow down access times and make navigation more confusing for users. By eliminating drop-down menus in favor of more visible navigation options, websites can improve the user experience and make it easier for users to find what they are looking for.

**Comment by bahadirgezer** [comment]
@bounswe/bounswe2023group8

**Comment by EgemenKaplan** [comment]
I can't just stand and say "No!" to this request after it's been backed up by such great arguments. I can only add that we should at least keep the drop-down label for the "Team members" part. It's one of our more redundant sections and it is absolutely not included in our vision of user experience.
Same could be said for meeting notes in the future because we will have dozens of them in a few months. We could archive notes older than 4 weeks under a drop-down and keep the new ones visible in the sidebar.

**Comment by bakikucukcakiroglu** [comment]
In my opinion, if someone thinks that something is better in any aspect, he/she should implement it without arguing a lot. Otherwise, these types of discussions may continue forever. That's why we have version control systems - we can revert any changes whenever we want. Even if we are against some ideas, we may miss a point that our teammate sees clearly. So, I support your idea since you think it is more useful for our team and can explain your reasoning.

**Comment by bahadirgezer** [comment]
Found a middle way: The details dropdown buttons are still available, but now they are open by default. Commit [link](https://github.com/bounswe/bounswe2023group8/wiki/_Sidebar/da1c362e6ac1e06515fe97a0417ba6d34b94e5f0)
